packages/bundle/elements.ts

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import '@spectrum-web-components/action-menu/sp-action-menu.js';
+import '@spectrum-web-components/actionbar/sp-actionbar.js';
+import '@spectrum-web-components/avatar/sp-avatar.js';
+import '@spectrum-web-components/banner/sp-banner.js';
+import '@spectrum-web-components/button/sp-button.js';
+import '@spectrum-web-components/button/sp-action-button.js';
+import '@spectrum-web-components/button/sp-clear-button.js';
+import '@spectrum-web-components/button-group/sp-button-group.js';
+import '@spectrum-web-components/card/sp-card.js';
+import '@spectrum-web-components/checkbox/sp-checkbox.js';
+import '@spectrum-web-components/circleloader/sp-circleloader.js';
+import '@spectrum-web-components/coachmark/sp-coachmark.js';
+import '@spectrum-web-components/dialog/sp-dialog.js';
+import '@spectrum-web-components/dialog/sp-dialog-wrapper.js';
+import '@spectrum-web-components/dropdown/sp-dropdown.js';
+import '@spectrum-web-components/dropzone/sp-dropzone.js';
+import '@spectrum-web-components/icon/sp-icon.js';
+import '@spectrum-web-components/icons/sp-icons-large.js';
+import '@spectrum-web-components/icons/sp-icons-medium.js';
```

Comment:
Did we want to include `icons-ui` and `icons-workflow` here?

Reply:
This may play into your suggestion to extend the documentation with this info, but this file is for side effectful imports, which means that were you to add `icons-*` exports here you would once again run into the problem of:
```
import '@spectrum-web-components/bundle/element.ts';
import { StarIcon } from '@spectrum-web-components/bundle/element.ts';
```
Allowing certain linters to error and or auto-delete one of the imports. As such, I think that it's important to continue to import icons from the default entry point:
```
import { StarIcon } from '@spectrum-web-components/bundle';
```
However, it's hard to say 100% as this package is specifically designed to surface _ALL_ of the component available in the larger project for prototyping efforts, without much consideration for perf, etc anyways...

Thoughts?

Reply:
Ah I see, that makes sense. Thanks for the examples! 

Even though `bundle` is designed to surface all components, it makes sense to me to limit the scope of the `bundle/element.ts` to side effect imports only. 

Running into linter errors while prototyping sounds like a pretty bad time haha